src/modules/auth/components/shared/google-auth-button/google-auth-button.tsx

```diff
@@ -1,9 +1,24 @@
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
 import { Box } from '@mui/material';
 import { ENDPOINTS } from '#/modules/auth/constants';
 import googleIcon from '#/modules/auth/assets/icons/google.svg';
 import styles from './google-auth-button.module.css';
+import { AccessTokenStore } from '#/utils';
 
 export function GoogleAuthButton() {
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const params = new URLSearchParams(window.location.search);
+    const accessToken = params.get('accessToken');
+
+    if (accessToken) {
+      AccessTokenStore.setToken(accessToken);
+      navigate('/');
+    }
```

Comment:
Wrap these into an async function to ensure the token is properly set before redirecting.

Reply:
removed this logic